generators/server/templates/dotnetcore/src/JHipsterNet/Pagination/SortExpressions.cs.ejs

```diff
@@ -0,0 +1,81 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+
+namespace JHipsterNet.Pagination
+{
+    public class SortExpressions<TEntity>
+    {
+        private readonly List<dynamic> _expressionList;
```

Comment:
You cannot use TEntity rather than dynamic ?

Reply:
no ! build error for List<TEntity>

Reply:
=/ i'm not really enthusiastic to use dynamic , maybe you can use TEntity,TKey ? Or if you have not other choose, maybe you can use object rather than dynamic

Reply:
I'll try to use `SortExpression<TEntity,TKey>` ratha than dynamic if it will work !

Reply:
done with SortExpression<TEntity, object> rather than dynamic